test/cpp_extensions/open_registration_extension/torch_openreg/torch_openreg/csrc/CMakeLists.txt

```diff
@@ -6,12 +6,49 @@ file(GLOB_RECURSE SOURCE_FILES
 
 add_library(${LIBRARY_NAME} SHARED ${SOURCE_FILES})
 
-target_link_libraries(${LIBRARY_NAME} PRIVATE torch_python torch_openreg)
+if(WIN32)
+    # For Windows, we need specific PyTorch libraries
+    find_library(TORCH_PYTHON_LIBRARY torch_python PATHS ${PYTORCH_INSTALL_DIR}/lib REQUIRED)
+    find_library(TORCH_CPU_LIBRARY torch_cpu PATHS ${PYTORCH_INSTALL_DIR}/lib REQUIRED)
+    find_library(C10_LIBRARY c10 PATHS ${PYTORCH_INSTALL_DIR}/lib REQUIRED)
+    
+    target_link_libraries(${LIBRARY_NAME} PRIVATE 
+        ${TORCH_PYTHON_LIBRARY}
+        ${TORCH_CPU_LIBRARY}
+        ${C10_LIBRARY}
+        torch_openreg
+    )
+else()
+    target_link_libraries(${LIBRARY_NAME} PRIVATE torch_python torch_openreg)
+endif()
```

Comment:
The cmake provided by PyTorch does not provide a cmake object like `torch_python_library`, so we need the above code to manually set the necessary information.

Can we encapsulate a torch_python_library cmake object to shield the above details? This object is necessary for third-party devices to implement the Python binding layer.